### ISSUE TITLE
generating an srt for an audio

### ISSUE THREAD

**Issue by wanjaswilly** [m1guelpf/auto-subtitle#16]
I would like to make a subtitle for an audio then generate an overlay video. can that be supported?

**Comment by binarykitchen** [comment]
Yup, that would be outstanding, a massive performance improvement when you already have an audio video separate to the video.
